src/Datadog.Trace.ClrProfiler.Managed.Core/InstrumentMethodAttribute.cs

```diff
@@ -11,8 +11,19 @@ public class InstrumentMethodAttribute : Attribute
     {
         /// <summary>
         /// Gets or sets the name of the assembly that contains the target method to be intercepted.
+        /// Required if <see cref="Assemblies"/> is not set.
         /// </summary>
-        public string Assembly { get; set; }
+        public string Assembly
+        {
+            get => string.Empty;
```

Comment:
Should this throw an exception or something? Seems wrong to use this, right?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1130/commits/de3ac0ca2cb72f6fdcbae97750c02af6bb1c65a7